src/style/app.module.css

```diff
@@ -2995,1128 +2641,2457 @@ form > input {
+}
+
+.opendrawer:hover {
+  transition: background-color 0.5s ease;
+  background-color: var(--bs-primary);
+}
+.collapseSidebarButton:hover {
+  transition: background-color 0.5s ease;
+  background-color: var(--bs-primary);
+}
+
+.customToggle {
+  padding: 0;
+  background: none;
+  border: none;
+  margin-right: 1rem;
+  --bs-btn-active-bg: none;
+}
+.customToggle svg {
+  color: black;
+}
+
+.customToggle::after {
+  content: none;
+}
+.customToggle:hover,
+.customToggle:focus,
+.customToggle:active {
+  background: none;
+  border: none;
+}
+.customToggle svg {
+  color: black;
+}
+
+.accordionBody {
+  height: calc(100vh / 2 - 2rem - 60px) !important;
+  overflow-y: scroll;
+}
+
+.accordionBody::-webkit-scrollbar {
+  display: none;
+}
+
+.filters {
+  padding: 20px 0px 0px 20px;
+  display: flex;
+  gap: 8px;
+}
+
+.filterButton {
+  border-radius: 14px;
+  padding: 5px 10px;
+  background-color: white;
+  color: #a5a5a5;
+  border: none;
+  border: 1px solid #a5a5a5;
+}
+
+.selectedBtn,
+.filterButton:hover {
+  border: 1px solid #98e0b6;
+  background-color: #98e0b6;
+  color: rgb(255, 255, 255);
+}
+
+.mainContainerDonate {
+  width: 50%;
+  flex-grow: 3;
+  display: flex;
+  flex-direction: column;
+  background-color: #f2f7ff;
+}
+
+.inputContainer {
+  flex: 1;
+  position: relative;
+}
+
+.input {
+  box-shadow: 5px 5px 4px 0px #31bb6b1f;
+}
+
+.box {
+  width: auto;
+  background-color: white;
+  margin-top: 1rem;
+  padding: 20px;
+  border: 1px solid #dddddd;
+  border-radius: 10px;
+}
+
+.heading {
+  font-size: 1.1rem;
+}
+
+.donationInputContainer {
+  display: flex;
+  flex-direction: row;
+  margin-top: 20px;
+}
+
+.donateBtn {
+  padding-inline: 1rem !important;
+}
+
+.dropdow {
+  min-width: 6rem;
+}
+
+.inputArea {
+  border: none;
+  outline: none;
+  background-color: #f1f3f6;
+}
+
+.maxWidthDonate {
+  width: 100%;
+}
+
+.donateActions {
+  margin-top: 1rem;
+  width: 100%;
+  display: flex;
+  flex-direction: row-reverse;
+}
+
+.donationsContainer {
+  padding-top: 4rem;
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+}
+
+.donationCardsContainer {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1rem;
+  --bs-gutter-x: 0;
+}
+
+.colorLight {
+  background-color: #f5f5f5;
+}
+
+.content {
+  padding-top: 10px;
+  flex-grow: 1;
+}
+
+.modal-dialog {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  margin: 0;
+}
+
+.modal-content {
+  width: 100%;
+  max-width: 500px;
+}
+
+.modal-body {
+  text-align: center;
+}
+
+.modal-header,
+.modal-footer {
+  justify-content: center;
+  text-align: center;
+}
+
+.confirmation {
+  text-align: center;
+  margin-top: 30px;
+}
+
+.contentOrg {
+  height: fit-content;
+  min-height: calc(100% - 40px);
+}
+
+.containerHeightOrg {
+  height: 100vh;
+  padding: 1rem 1.5rem 0 calc(300px + 1.5rem);
+}
+
+.colorPrimary {
+  background: #31bb6b;
+}
+
+.input {
+  margin-right: 10px;
+}
+
+.borderBox {
+  border: 1px solid #dddddd;
```

Comment:
Use a variable. There are other similar colors defined at the top of the file you don't need to redefine one.